pkg/distributor/ha_tracker.go

```diff
@@ -58,7 +58,8 @@ type HATrackerConfig struct {
 	// other than the replica written in the KVStore if the difference
 	// between the stored timestamp and the time we received a sample is
 	// more than this duration
-	FailoverTimeout time.Duration `yaml:"ha_tracker_failover_timeout"`
+	FailoverTimeout    time.Duration `yaml:"ha_tracker_failover_timeout"`
+	CleanupOldReplicas bool          `yaml:"cleanup_old_replicas"`
```

Comment:
Any reason why we don't want to always keep it enabled?

Reply:
My thinking was to test it first in our production (where we have some of these old HA pairs), and then enable it and remove this option. WDYT?

Reply:
I'm fine as far as we remove it once we're confident. It's marked experimental, so we can remove it later.

Reply:
Thinking about it more, that adds burden to our deployment. I'm inclined to remove it.

Reply:
Discussed offline, decided to remove, as it's quick to roll back distributors in case there are problems with this feature.